services/data-record-service.js

```diff
@@ -827,6 +815,79 @@ class DataRecordService {
         return await this.dataRecordsCollection.distinct("nodeType", filter);
     }
 
+    #getSubmissionStatQuery(submissionID, validNodeStatus) {
+        return [
+            {$match:{
+                    submissionID: submissionID,
+                    status: {$in: validNodeStatus}
+            }},
+            {$group:{
+                    _id: {submissionID: "$submissionID"},
+                    fileNames: {$addToSet: "$s3FileInfo.fileName"},
```

Comment:
Since the file names list is split into a separate aggregation, I think the fileNames property here can be removed